src/components/DesignSytem/DesignSystem.module.scss

```diff
@@ -0,0 +1,122 @@
+/* Primary Color*/
+$cyanDark: #2B788B;
+$cyanLight: #C3DCE3;
+$pinkDark: #945069;
+$pinkLight: #F2D4DC;
+$black: #000000;
+$greyLight: #F6F5F4;
+$greyMedium: #E0E0E0;
+$greyIcon: #BABABA;
+$greyDark: #757575;
+
+/* Secondary Color*/
+$cyan: #5996A5;
+$green: #639B6D;
+$pink: #A15993;
+$red: #A95151;
+$yellow: #C4A24C;
+$orange: #CB5B43;
+
+.designContainer {
+    display: flex;
+    gap: 100px;
+  }
+
+.headingColumn{
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
Do you think these should be inside of the design system (theme style) or they are only for specific pages?